tests/phpunit/CRM/Core/BAO/FinancialTrxnTest.php

```diff
@@ -141,40 +148,29 @@ public function testCreateDeferredTrxn() {
       'receive_date' => '2016-01-20',
       'total_amount' => 622,
       'financial_type_id' => 4,
-      'line_items' => [
```

Comment:
These are for $100 but cont total is 622 = invalid data

Reply:
LOL